### PR TITLE
feat: skeleton docs

### PR DESCRIPTION
**Pull request by danielrobertson**
Closes # 

## 📝 Description

Add skeleton to docs website 

## Screenshots

image

## Merge checklist

- [ ] Added/updated tests
- [ ] Added changeset

### CODE REVIEW

File: apps/website/docs/components/skeleton.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+Title: Skeleton
```
Comment (bclark-p44) [inline apps/website/docs/components/skeleton.mdx:2]: lower case `title` or the nav wont pick it up.

File: apps/website/docs/components/skeleton.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+## Best Practices
+
+**When to use**
+
+When you need to prevent Cumulative Layout Shift (CLS) or provide a visual placeholder for an
+element that is loading.
+
+## Content
+
+## Accessibility
```
Comment (bclark-p44) [inline apps/website/docs/components/skeleton.mdx:19]: lets remove best pracitces, content and accessibility for now. Follow this pattern:

<title>
<description>
<demo>
Examples
<examples>
Props
<props_table>

File: apps/website/docs/components/avatar.mdx [hunk 3]
```diff
@@ -11,7 +11,7 @@ An avatar is visual representation of a user or entity.
 
 **When to use**
```
Comment (bclark-p44) [inline apps/website/docs/components/avatar.mdx:13]: while you're in here, could you fix the `title` casing issue as well?

File: apps/website/docs/components/skeleton.mdx (hunk 2 above)
Comment (danielrobertson) [reply apps/website/docs/components/skeleton.mdx]: > lets remove best pracitces, content and accessibility for now. Follow this pattern:
> 
> <title> Examples Props

Removed those headers. Unsure what you mean by `<title> Examples Props` pattern however. Take a look now to see if I should make additional updates!

**Pull request merged**